ajax/libs/ng-dropzone/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ng-dropzone",
+  "filename": "ng-dropzone.min.js",
+  "description": "AngularJS directive for __[dropzone](https://github.com/enyo/dropzone)__",
+  "homepage": "http://www.dropzonejs.com",
+  "keywords": [
+    "ng-dropzone"
```

Comment:
Add a few more keywords for easy discovery:
* angularjs
* directive
* drag
* drop
* file
* upload